src/Modal/Modal.js

```diff
@@ -0,0 +1,73 @@
+/** @jsx jsx */
+import ReactModal from 'react-modal';
+import PropTypes from 'prop-types';
+import omit from 'lodash/omit';
+import { Fragment } from 'react';
+import { Global, css, jsx } from '@emotion/core';
+import { transparentize } from 'polished';
+import { withTheme } from 'emotion-theming';
+
+const baseOverlayCss = theme => css`
+  align-items: center;
+  background: ${transparentize(0.72, theme.colors.grey100)};
+  bottom: 0;
+  display: flex;
+  justify-content: center;
+  left: 0;
+  position: fixed;
+  right: 0;
+  top: 0;
+`;
+
+const baseModalCss = theme => css`
```

Comment:
What do you think about pulling this and `baseOverlayCss` out into a `styles` file within `Modal?

Reply:
I'd be down with that. Clean it up a bit.